{
  try {
    validator.validate(new ThrowsException(null),null);
    fail();
  }
 catch (  MockitoException e) {
  }
}
